garnett/dcdfilereader.py

```diff
@@ -180,17 +184,18 @@ def read(self):
             raw_frame.data_keys = copy.deepcopy(self.t_frame.data_keys)
             raw_frame.box_dimensions = self.t_frame.box.dimensions
             try:
-                raw_frame.shapedef = copy.deepcopy(self.t_frame.shapedef)
+                raw_frame.type_shapes = copy.deepcopy(self.t_frame.type_shapes)
             except AttributeError:
                 pass
         if not self._loaded():
             self._load()
         assert self._loaded()
         raw_frame.box = self._box
-        raw_frame.types = copy.copy(self._types)
+        raw_frame.types = self._types
```

Comment:
It's not clear to me why the `copy.copy` was required before. It may not be needed?